src/models/activity_pub/object.cr

```diff
@@ -606,8 +606,8 @@ module ActivityPub
           Tag::Mention.new(name: name, href: href)
         end,
         attachments: dig_values?(json, "https://www.w3.org/ns/activitystreams#attachment") do |attachment|
-          url = attachment.dig?("https://www.w3.org/ns/activitystreams#url").try(&.as_s)
-          media_type = attachment.dig?("https://www.w3.org/ns/activitystreams#mediaType").try(&.as_s)
+          url = attachment["url"].try(&.as_s)
+          media_type = attachment["mediaType"].try(&.as_s)
```

Comment:
These changes were necessitated by the change to `ld_json.cr`. The two things labeled attachment don't have the same schema. I think the Mastodon implementation on `actor` is non-standard. The activitiystreams context has
```
    "attachment": {
      "@id": "as:attachment",
      "@type": "@id"
    },
```

That doesn't match what Mastodon uses for actors.

If there is a better way to implement this, let me know. This is what I could figure out.

Reply:
this definition is okay. it's defining attachment (both for actors and objects) as the activity streams ("as") attachment. the missing piece is the definition of the PropertyValue type and value. if you make the other changes i suggest, you should be able to get rid of this change.